templates/default/less/variables.less

```diff
@@ -90,16 +93,23 @@
 @border-radius-large:       0px;
 @border-radius-small:       0px;
 
+// This controls the heights of the header components.
+// controls height of the "topbar" (that gray thingy on top of the page in the standard).
+@il-top-bar-height:			40px;
+// controls the height of the "main header" (that white thingy with the logo and the main menu in it).
+@il-main-header-height:		80px;
```

Comment:
Don't really know either, this is a question to the KS, i think. The names try to mimic the css-class names, so it's best we could think of.